### PR TITLE
change role

### PR DESCRIPTION
**Pull request by sunghwan-park-fw**

### CODE REVIEW

File: apps/web/src/components/Keycloak.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ const withAuth = (Component: React.FunctionComponent, roles: ValidRoles[]) => {
       if (!authUser && !authUserLoading && kc.initialized && !kc?.keycloak?.authenticated) {
         router.replace('/login');
       }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (dbayly-freshworks) [inline apps/web/src/components/Keycloak.tsx:26]: We might want to create a tech debt ticket to look into exhaustive deps.

File: apps/web/src/components/display/AddRecordModal.tsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ export const AddRecordModal: React.FC<AddRecordProps> = (props: AddRecordProps)
   };
 
   return (
-    <Modal open={visible} handleClose={close}>
+    <Modal open={visible} handleClose={() => void 0}>
```
Comment (dbayly-freshworks) [inline apps/web/src/components/display/AddRecordModal.tsx:90]: I'm curious about whats going on here. How does this function work?

**Review by dbayly-freshworks — APPROVED** [review]
This all looks good to me, I just have two small questions. I am really glad ValidRoles is getting shifted to common.

**Review by bhumin-fw — APPROVED** [review]
It looks good

File: apps/web/src/components/display/AddRecordModal.tsx (hunk 2 above)
Comment (sunghwan-park-fw) [reply apps/web/src/components/display/AddRecordModal.tsx:90]: it's same as `undefined`. Modal requires handleClose prop but it isn't called ever and I don't know why.

**Pull request merged**